apps/sms/test/marionette/lib/messages.js

```diff
@@ -431,6 +440,21 @@
 
             window.wrappedJSObject.TestStorages.setStorageReady('messagesDB');
           }, [threads || [], uniqueMessageIdCounter || 0]);
+        },
+
+        /**
+         * Sets pre-populated contacts storage.
+         * @param {Array.<MozContact>?} contacts List of the contacts to
+         * pre-populate storage with.
+         */
+        setContactsStorage: function(contacts) {
```

Comment:
> should we rename previous setStorage to setMessagesStorage ?

Mm yes, why not.

> also I wonder if these 2 functions should not be in a separate js file...

Didn't get, you want to have separate js file just for this method?